content/docs/components/ksonnet.md

```diff
@@ -44,53 +44,33 @@ Follow the steps below to install ksonnet:
-
-```
-export KUBEFLOW_VERSION=0.2.2
-export KUBEFLOW_KS_DIR=</path/to/store/your/ksonnet/application>
-export KUBEFLOW_DEPLOY=false
-curl https://raw.githubusercontent.com/kubeflow/kubeflow/v${KUBEFLOW_VERSION}/scripts/deploy.sh | bash
-```
-
-This will create a ksonnet application in ${KUBEFLOW_KS_DIR}. Refer to [deploy.sh](https://github.com/kubeflow/kubeflow/blob/v0.2-branch/scripts/deploy.sh)
-to see the individual commands run.
-
-**Important**: The commands above will enable collection of **anonymous** user data to help us improve Kubeflow. Do disable usage collection you
-can run the following commands
-
-```
-cd ${KUBEFLOW_KS_DIR}
-ks param set spartakus reportUsage false
-```
-
-You can now deploy Kubeflow as follows
-
-```
-cd ${KUBEFLOW_KS_DIR}
-ks apply default
-```
+After installing ksonnet, you can follow the 
+[Kubeflow getting-started guide](/docs/started/getting-started) to deploy
+Kubeflow.
 
 ## Upgrading ksonnet
 
 See the guide to [upgrading Kubeflow](/docs/other-guides/upgrade/).
 
 ## Why Kubeflow uses ksonnet
 
-ksonnet makes it easier to manage complex deployments consisting of multiple components. It is designed to
-work side by side with kubectl.
+ksonnet makes it easier to manage complex deployments consisting of multiple
+components. ksonnet is designed to work side by side with `kubectl`.
 
-ksonnet allows us to generate Kubernetes manifests from parameterized templates. This makes it easy to customize Kubernetes manifests for your
-particular use case. In the examples above we used this functionality to generate manifests for TfServing with a user supplied URI for the model.
+ksonnet allows you to generate Kubernetes manifests from parameterized
+templates. This makes it easy to customize Kubernetes manifests for your
+particular use case.
 
-One of the reasons we really like ksonnet is because it treats [environment](https://ksonnet.io/docs/concepts#environment) as in (dev, test, staging, prod) as a first class concept. For each environment we can easily deploy the same components but with slightly different parameters
-to customize it for a particular environments. We think this maps really well to common workflows. For example, this feature makes it really
-easy to run a job locally without GPUs for a small number of steps to make sure the code doesn't crash, and then easily move that to the
-Cloud to run at scale with GPUs.
+ksonnet treats [environment](https://ksonnet.io/docs/concepts#environment)
+as a first class concept. For example, you can define separate
+environments for your development, test, staging, and production deployments.
+For each environment, you can use ksonnet to deploy the same components 
+with slightly different parameters to customize the deployment for a particular 
+environment. For example, this feature makes it easy to run a job locally 
+without GPUs for a small number of steps to make sure the code doesn't crash, 
+then move the deployment to the Cloud to run at scale with GPUs.
```

Comment:
I don't think cloud should be capitalized, since it's not a proper name. cloud.google.com and kubernetes.io both use "the cloud" (lowercase). You could also consider "cloud infrastructure" or another term.

Reply:
Done.